rosdep/python.yaml

```diff
@@ -2119,6 +2119,19 @@ python-libpgm-pip:
   ubuntu:
     pip:
       packages: [libpgm]
+python-librabbitmq-pip:
+  debian:
+    pip:
+      packages: [librabbitmq]
```

Comment:
You should add a depends tag onto the c implementation so that rosdep knows to install it first via apt.  Here and the other 2 distros below too.